include/tinyalsa/plugin.h

```diff
@@ -128,6 +129,11 @@ struct pcm_plugin_ops {
     /** Any custom or alsa specific ioctl implementation */
     int (*ioctl) (struct pcm_plugin *plugin,
                   int cmd, void *arg);
+    void* (*mmap) (struct pcm_plugin *plugin, void *addr, size_t length,
```

Comment:
Minor issue, but to match Linux kernel style, please move * one place to the right